app/models/ability.rb

```diff
@@ -5,6 +5,7 @@ def initialize(user)
     user ||= User.new
 
     # Abilities that everyone get.
+    can :index, Tool
```

Comment:
It is not needed, however the `:modify` is only called on the Admin-controller.
If we would want to restrict the access to the normal index method it would be in:
https://github.com/fsek/web/pull/354/files#diff-5029dd0b0c3115d1881efa5141a28fc7R3